config/prow/plugins.yaml

```diff
@@ -226,36 +226,39 @@ slack:
     - k8s-ci-robot # future home of tide
     - k8s-release-robot # anago
     branch_whitelist:
-        release-1.14:
+        release-1.15:
             - aleksandra-malinowska # Patch Release Team
             - dougm # Patch Release Team
             - feiskyer # Patch Release Team
             - hoegaarden # Patch Release Team
             - idealhack # Patch Release Team
             - tpepper # Patch Release Team
-        release-1.15:
+        release-1.16:
             - aleksandra-malinowska # Patch Release Team
             - dougm # Patch Release Team
             - feiskyer # Patch Release Team
             - hoegaarden # Patch Release Team
             - idealhack # Patch Release Team
             - tpepper # Patch Release Team
-        release-1.16:
+        release-1.17:
             - aleksandra-malinowska # Patch Release Team
+            - bubblemelon # Branch Manager
+            - cpanato # Branch Manager
```

Comment:
Remove. Only Patch Release Team members should be listed on the patch release branches.

Reply:
done